src/libraries/AccountAccessParser.sol

```diff
@@ -541,6 +541,7 @@ library AccountAccessParser {
     ) internal view noGasMetering {
         console.log("");
         string memory line = unicode"━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━";
+        printCopyHelper("below");
```

Comment:
I moved this here because I actually want users to copy any value transfers into the VALIDATIONS file too. 
![CleanShot 2025-05-15 at 13 59 49@2x](https://github.com/user-attachments/assets/e1237205-63e5-4060-9988-3626a364a4e8)

This change is minimally invasive and I'm happy to include it in this PR.